File: eMusicAngular/src/app/models/track.ts
export class Track {
    id: string;
    trackPath: string;
    imagePath: string;
    artist: string;
    title: string;

    constructor(id: string, trackPath: string, imagePath: string, artist: string, title: string) {
        this.id = id;
        this.trackPath = trackPath;
        this.imagePath = imagePath;
        this.artist = artist;
        this.title = title;
    }
}

File: eMusic.API/eMusic.API/Models/Track.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace eMusic.API.Models
{
    public class Track
    {
        public int Id { get; set; }
        public string Artist { get; set; }
        public string Title { get; set; }
        public string TrackPath { get; set; }
        public string ImagePath { get; set; }
        public decimal Cost { get; set; }

        [JsonIgnore]
        public List<UserTrack> UserTracks { get; set; }
    }
}
